python/examples/minimal_build/README.md

```diff
@@ -24,38 +24,42 @@ an eye toward maximum portability across different Linux distributions. This
 may help for contributors debugging build issues caused by their local
 environments.
 
-## Fedora 31
+## Ubuntu 18.04
 
 Build image:
 
 ```
-docker build -t arrow_fedora_minimal -f Dockerfile.fedora
+docker build -t arrow_ubuntu_minimal -f Dockerfile.ubuntu
 ```
 
 Build with conda or pip/virtualenv:
 
 ```
 # With pip/virtualenv
-docker run --rm -t -i -v $PWD:/io arrow_fedora_minimal /io/build_venv.sh
+docker run --rm -t -i -v $PWD:/io arrow_ubuntu_minimal /io/build_venv.sh
 
 # With conda
-docker run --rm -t -i -v $PWD:/io arrow_fedora_minimal /io/build_conda.sh
+docker run --rm -t -i -v $PWD:/io arrow_ubuntu_minimal /io/build_conda.sh
 ```
 
-## Ubuntu 18.04
+## Fedora 31
+
+Fedora uses podman instead of docker. In case of issues please first ensure
+they are not podman related - test setup with standard docker first
+with commands with from the Ubuntu section.
 
 Build image:
 
 ```
-docker build -t arrow_ubuntu_minimal -f Dockerfile.ubuntu
+podman build -t arrow_fedora_minimal -f Dockerfile.fedora
 ```
 
 Build with conda or pip/virtualenv:
 
 ```
 # With pip/virtualenv
-docker run --rm -t -i -v $PWD:/io arrow_ubuntu_minimal /io/build_venv.sh
+podman run --rm -i -t arrow_fedora_minimal /io/build_venv.sh
```

Comment:
Doesn't `podman` support the `-v` option to create a volume? Is this why you added the `COPY` lines to the dockerfile?

Reply:
It does but it is privilaged operation so you either need to disable security or configure it properly.

Copying scripts at the end of docker file is simpler.

But this is probably for separate section...